addons/website_links/static/src/js/website_links.js

```diff
@@ -533,6 +535,27 @@ publicWidget.registry.websiteLinks = publicWidget.Widget.extend({
     },
 });
 
+popupWidget.include({
+    /**
+     * @override
+     */
+    _onHideModal() {
```

Comment:
Hooking yourself on this `_onHideModal` will not exactly work. You are hooking on any popup closing, even non cookies ones.
Eg, closing this one will also set the utm in cookies, which is wrong
![image](https://github.com/odoo/odoo/assets/30048408/44e45899-f851-4e49-af4b-5f8ca33fd834)

Maybe it's just about including `publicWidget.registry.cookies_bar` instead of `popupWidget`. I'll let you have a look